.github/workflows/update-examples-cron.yml

```diff
@@ -0,0 +1,27 @@
+name: Update Examples Cron
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-examples-cron:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - run: |
+          make tools/update_examples
+          if ! git diff --exit-code;
+          then
+            make tools/update_examples_pr
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Unsure if this is the right token, this does something very similar: https://github.com/grafana/pyroscope/blob/9b5c6d7f8c7eadf9bcb596c4b0ffc3f5b5607ddb/.github/workflows/update-contributors.yml#L32